roles/cifmw_cephadm/tasks/configure_object.yml

```diff
@@ -83,8 +83,6 @@
   when:
     - cifmw_cephadm_certificate | length > 0
     - cifmw_cephadm_key | length > 0
-    - cert_stat_result.stat.exists | bool
-    - key_stat_result.stat.exists | bool
```

Comment:
These stat result vars would be out of date with the current state of the system because they are made before the certificate is created. 

`rgw.yml` calls the new `cifmw_external_dns` role which creates the certificate whenever `cifmw_cephadm_key` and `cifmw_cephadm_certificate` have `length > 0`. 

If we got to this point, then the new role didn't fail the certificates have  been created.